content/francais/python-turtle/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Python : Coder avec Turtle"
+description: "Apprenez à dessiner sur un canevas avec Python et Turtle"
+date: 2024-09-17T13:24:17-07:00
+prereq: "Bases de Python"
+difficulty: "Débutant"
+image: ""
+draft: false
+icon: "fab fa-python"
+---
+
+## Introduction
+
+<p style="text-align: center;"><iframe width="560" height="315" src="https://www.youtube.com/embed/uw8_cEooILM" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe></p>
+
+Bienvenue à la Nouvelle Ruche ! Vous arrivez juste à temps pour aider notre nouvelle abeille Alex.
+
+![animation de la jeune abeille Alex](https://media1.giphy.com/media/ozjz5omKqJYex8CaDV/giphy.gif)
+
+Alex vit dans un pays nommé les Îles Tortues. Les habitants des Îles Tortues parlent une langue appelée Python. Nous allons utiliser Python pour apprendre à Alex comment vivre dans une ruche. Prêt pour le défi ? Dans ce tutoriel, nous apprendrons à utiliser le langage Python pour aider Alex à *devenir* une abeille bien occupée !
+
+## Table des Matières
+<details>
+<summary>Sections</summary>
```

Comment:
Sections should be translated... but this is a minefield to navigate around the HTML tags :P

Reply:
haha true prompts is confusing the LLM models. some sections still needs manual corrections and updates.